todo/src/reducers/index.js

```diff
@@ -0,0 +1,27 @@
+import { ADD_TODO, TOGGLE_TODO } from '../actions' 
+
+const initialState = {
+    todos: []
+};
+
+export default (state = initialState, action) => {
+    switch(action.type){
+        case ADD_TODO: 
+        return Object.assign({}, state, {
```

Comment:
Same as
```suggestion
        return { ...state
```
for a more modern solution